analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/ProgramStateTest.CaptureOperation.cs

```diff
@@ -106,4 +106,23 @@ public void ResolveCapture_OtherOperation_ReturnsArgument()
         var sut = ProgramState.Empty.SetCapture(capture.Id, capture.Value);
         sut.ResolveCapture(capture.Value).Should().Be(capture.Value);   // Any other operation
     }
+
+    [TestMethod]
+    public void ResolveCaptureAndUnwrapConversion_CaptureReferenceInConversion_ReturnsCapturedOperation()
```

Comment:
You'll need and UT for capture reference that was thrown away => add UT with captures-only, add PostProcessTestCheck that will register for `FlowCapture` and will return `State.Empty` to throw away captures.